Plugins/PluginUtilities/DynPacket.h

```diff
@@ -0,0 +1,270 @@
+#include "FLHook.h"
+
+// Use the class to create and send packets of inconstant size.
+#pragma pack(push, 1)
+class FLPACKET
+{
+private:
+	uint Size;
+
+	byte kind;
+	byte type;
+
+public:
+	// This is content of your packet. You may want to reinterpretate it as pointer to packet data struct for convenience.
+	byte content[1];

```

Comment:
I'm struggling to get my head around the `[1]` here. Shouldn't it just be a pointer?

Reply:
We need [1] as flexible array member to make packet struct contiguous in memory.
Pointer would split packet into two allocated memory zones.